checkout-core/src/main/java/com/adyen/checkout/core/internal/data/model/JsonUtils.kt

```diff
@@ -54,6 +59,38 @@ fun JSONArray.toStringPretty(): String {
     }
 }
 
+@RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
+@Throws(JSONException::class)
+inline fun <reified T : ModelObject> JSONObject.jsonToMap(
```

Comment:
What's the difference between these 2 methods?

Reply:
hmm just wanted to separate the logic, but we can combine them.

Reply:
We can separate them as long as we give them clear different names to explain what each one does, and make one of them private since it's not supposed to be used directly